studio/src/components/dashboard/workspace-provider.tsx

```
@@ -0,0 +1,106 @@
+    () => data?.namespaces.reduce(
+      (acc, wns) => {
+        acc.set(wns.name, wns);
+        return acc;
+      },
+      new Map<string, WorkspaceNamespace>(),
+    ) ?? new Map<string, WorkspaceNamespace>(),
+    [data?.namespaces],
+  );
+
+  const setNamespaceCallback = useCallback((ns: string, applyRouteParams: boolean) => {
+    if (!ns || namespace === ns || !namespaces.some((ns) => ns.toLowerCase() === ns.toLowerCase())) {
```

Biome (2.1.2)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)